sympy/geometry/line.py

```diff
@@ -623,12 +687,14 @@ def is_perpendicular(l1, l2):
         >>> l3 = Line(p1, p4)
         >>> l1.is_perpendicular(l3)
         False
+
+        perpendicularity over 3D entities.
         >>> from sympy import Point3D, Line3D
         >>> p1, p2, p3 = Point3D(0, 0, 0), Point3D(1, 1, 1), Point3D(-1, 2, 0)
         >>> l1, l2 = Line3D(p1, p2), Line3D(p2, p3)
         >>> l1.is_perpendicular(l2)
         False
-        >>> p4 = Point3D(5, 3, 7)
+        >>> p4 = Point3D(1, 1, -1)
```

Comment:
Any reason for changing this example?

Reply:
I actually wanted to show both the answer: true and false
I will change the p4 point to (1, 1, -2).